src/components/Header.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+/**
+ * App title .
+ */
+const Header = () => <h1>Hacker News</h1>;
```

Comment:
``` <h1> {appContants.HEADER} </h1> ```

You might have used Hacker News in multiple places. What if someone asked you to change your application name form Hacker New to Hacker Post ?